synapse/tests/test_model_orgs.py

```diff
@@ -1,52 +1,180 @@
+                self.eq(node.get('naics'), '541715')
+                self.eq(node.get('url'), 'http://www.arrowinc.link')
+                self.eq(node.get('us:cage'), '7qe71')
+
+                # ou:suborg
+                guid1 = s_common.guid()
+                subprops = {
+                    'perc': 50,
+                    'current': True,
+                    # Fixme - seen:min / seen:max
+                }
+                node = xact.addNode('ou:suborg', (guid0, guid1), subprops)
+                self.eq(node.ndef[1], (guid0, guid1))
+                self.eq(node.get('perc'), 50)
+                self.eq(node.get('current'), 1)
+                self.raises(s_exc.BadTypeValu, node.set, 'perc', 101)
+                self.raises(s_exc.BadTypeValu, node.set, 'perc', 0)
+
+                # ou:user
+                node = xact.addNode('ou:user', (guid0, 'arrowman'))
+                self.eq(node.ndef[1], (guid0, 'arrowman'))
+                self.eq(node.get('org'), guid0)
+                self.eq(node.get('user'), 'arrowman')
+
+                # ou:haslias
+                node = xact.addNode('ou:hasalias', (guid0, 'EVILCORP'))
+                self.eq(node.ndef[1], (guid0, 'evilcorp'))
+                self.eq(node.get('alias'), 'evilcorp')
+                self.eq(node.get('org'), guid0)
+
+                # ou:org:has
+                # FIXME this should be a test form eventually
+                node = xact.addNode('ou:org:has', (guid0, ('teststr', 'pretty floral bonnet')))
+                self.eq(node.ndef[1], (guid0, ('teststr', 'pretty floral bonnet')))
+                self.eq(node.get('org'), guid0)
+                self.eq(node.get('node'), ('teststr', 'pretty floral bonnet'))
+                self.eq(node.get('node:form'), 'teststr')
+                # FIXME This is an autoadds test and should be tested elsewhere
+                nodes = list(xact.getNodesBy('teststr', 'pretty floral bonnet'))
+                self.len(1, nodes)
+                # FIXME seen:min / seen:max
+
+                # ou:meet
+                m0 = s_common.guid()
+                mprops = {
+                    'name': 'Working Lunch',
+                    'start': '201604011200',
+                    'end': '201604011300'
+                    # 'place': '', # FIXME geospatial
+                }
+                node = xact.addNode('ou:meet', m0, mprops)
+                self.eq(node.ndef[1], m0)
+                self.eq(node.get('name'), 'working lunch')
+                self.eq(node.get('start'), 1459512000000)
+                self.eq(node.get('end'), 1459515600000)
+
+                # ou:conference
+                c0 = s_common.guid()
+                cprops = {
+                    'org': guid0,
+                    'name': 'arrowcon 2018',
+                    'base': 'arrowcon',
+                    'start': '20180301',
+                    'end': '20180303',
+                    # 'place': '', # FIXME geospatial
```

Comment:
`geo:place` is available now

Reply:
we don't have any tests in place for ``geo:place``

Reply:
I was wrong, `geo:place` type was included by mistake. Re-commented it out in my PR until guid aliases exist again